Assets/Scripts/WackyTappy/Squished.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using static SlimeSquishedGameController;
+
+public class Squished : MonoBehaviour
```

Comment:
I'd rename this to `SquishableComponent` to better convey that objects that have this component have this "squishable" functionality, and will run this code when they are updated c: